store/user.js

```diff
@@ -0,0 +1,34 @@
+const state = () => ({
+  adminData: null,
+})
+
+const getters = {
+  getAdmin(state){
+    return state.adminData
+  },
+  isAdmin(state){
+    return !!state.adminData
+  }
+}
+
+const mutations = {
+  setAdmin(state, user){
```

Comment:
Not sure if having the params adjacent like that work. Usually it is a nested callback i.e. 
```
state => user => {}
```

Reply:
It do be working